### PR TITLE
V14 RC: Update documentation for Two-Factor Authentication

### PR DESCRIPTION
**Pull request by iOvergaard**
## Description

Added/Updated documentation needed for Two-Factor Authentication with Umbraco 14.

## Type of suggestion

* [x] Updates related to a new version

## Product & version (if relevant)
14.0.0

## Deadline (if relevant)

ASAP -- this is already in effect in RC3

### CODE REVIEW

File: 14/umbraco-cms/reference/security/two-factor-authentication.md [hunk 1]
```diff
@@ -253,15 +254,15 @@ public class UmbracoUserAppAuthenticator : ITwoFactorProvider
     /// <param name="userOrMemberKey">The key of the user or member</param>
     /// <param name="secret">The secret that ensures only this user can connect to the authenticator app</param>
     /// <returns>The required data to setup the authenticator app</returns>
-    public Task<object> GetSetupDataAsync(Guid userOrMemberKey, string secret)
+    public Task<ISetupTwoFactorModel> GetSetupDataAsync(Guid userOrMemberKey, string secret)
     {
         IUser? user = _userService.GetByKey(userOrMemberKey);
```
Comment (alina-tincas) [inline 14/umbraco-cms/reference/security/two-factor-authentication.md:259]: Getting a warning about this, is it ok to leave this as it is?
image

File: 14/umbraco-cms/reference/security/two-factor-authentication.md [hunk 2]
```diff
@@ -206,11 +206,12 @@ using Google.Authenticator;
 using Umbraco.Cms.Core.Models.Membership;
 using Umbraco.Cms.Core.Security;
 using Umbraco.Cms.Core.Services;
+using Umbraco.Extensions;
```
Comment (alina-tincas) [inline 14/umbraco-cms/reference/security/two-factor-authentication.md:209]: it also shows that this not necessary to be added as is it not used anywhere 🤔 
image

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Looks good, merging this in 💪 Will also make this article visible on docs.

File: 14/umbraco-cms/reference/security/two-factor-authentication.md (hunk 2 above)
Comment (iOvergaard) [reply 14/umbraco-cms/reference/security/two-factor-authentication.md:209]: I think you are right. That can go away. My IDE messed up!

**Pull request merged**